card/mod/05_standard/set/self/head.rb

```diff
@@ -13,79 +12,93 @@
+  end
+
+  def rss_link
+    opts = { format: :rss }
+    root.search_params[:vars].each { |key, val| opts["_#{key}"] = val }
+    href = page_path root.card.cardname, opts
+    %{<link rel="alternate" type="application/rss+xml" title="RSS" href="#{href}" />}
   end
 
   def head_stylesheets
     manual_style = params[:style]
-    style_card   = Card[manual_style] if manual_style
+    style_card = Card[manual_style] if manual_style
     style_card ||= root.card.rule_card :style
-    @css_path = if params[:debug] == 'style'
-      page_path( style_card.cardname, item: :import, format: :css)
-    elsif style_card
-      card_path style_card.machine_output_url
-    end
-
-    if @css_path
-      %{<link href="#{@css_path}" media="all" rel="stylesheet" type="text/css" />}
-    end
+    @css_path =
+      if params[:debug] == 'style'
+      page_path(style_card.cardname, item: :import, format: :css)
+      elsif style_card
+        card_path style_card.machine_output_url
+      end
+    return unless @css_path
+    %{<link href="#{@css_path}" media="all" rel="stylesheet" type="text/css" />}
   end
 
   def head_javascript
     varvals = [
       "window.wagn={rootPath:'#{ Card.config.relative_url_root }'}",
-      "window.tinyMCEPreInit={base:\"#{card_path 'assets/tinymce'}\",query:'3.5.9',suffix:''}" # tinyMCE doesn't load on non-root wagns w/o preinit line
+      # tinyMCE doesn't load on non-root wagns w/o preinit line
+      "window.tinyMCEPreInit={base:\"#{card_path 'assets/tinymce'}\",query:'3.5.9',suffix:''}"
     ]
-    card.have_recaptcha_keys?                        and varvals << "wagn.recaptchaKey='#{Card.config.recaptcha_public_key}'"
-    c=Card[:double_click] and !Card.toggle c.content and varvals << 'wagn.noDoubleClick=true'
-    @css_path                                        and varvals << "wagn.cssPath='#{@css_path}'"
+    card.have_recaptcha_keys? &&
+      varvals << "wagn.recaptchaKey='#{Card.config.recaptcha_public_key}'"
+    (c = Card[:double_click]) && !Card.toggle(c.content) &&
+      varvals << 'wagn.noDoubleClick=true'
+    @css_path &&
+      varvals << "wagn.cssPath='#{@css_path}'"
 
     manual_script = params[:script]
     script_card   = Card[manual_script] if manual_script
     script_card ||= root.card.rule_card :script
 
-    @js_tag = if params[:debug] == 'script'
-      script_card.format(:js).render_core item: :include_tag
-    elsif script_card
-      javascript_include_tag script_card.machine_output_url
-    end
-    ie9_card    = Card[:script_html5shiv_printshiv]
-    %(#{ javascript_tag do varvals * ';' end  }
+    @js_tag =
+      if params[:debug] == 'script'
+        script_card.format(:js).render_core item: :include_tag
+      elsif script_card
+        javascript_include_tag script_card.machine_output_url
+      end
+    ie9_card = Card[:script_html5shiv_printshiv]
+    <<-HTML
+      #{ javascript_tag do varvals * ';' end  }
```

Comment:
Prefer `{...}` over `do...end` for single-line blocks.<br>Space inside string interpolation detected.